static/app/views/projectInstall/createProject.tsx

```diff
@@ -230,12 +230,6 @@ function CreateProject() {
       ),
       {
         modalCss,
-        onClose: () => {
```

Comment:
don't we want to fire the analytics event when users click on the backdrop of a modal, closing it?

Reply:
This was really good feedback. I have fixed this by removing calling the `onClose` callback when "Skip" and "Configure SDK" buttons are called